server/model/monitor.js

```diff
@@ -447,6 +456,18 @@ class Monitor extends BeanModel {
                         options.data = bodyValue;
                     }
 
+                    if (this.cacheBust) {
+                        let cacheBust = "";
+                        const characters = "abcdefghijklmnopqrstuvwxyz0123456789";
+                        const charactersLength = characters.length;
+                        for (let i = 0; i < 6; i++) {
+                            cacheBust += characters.charAt(Math.floor(Math.random() * charactersLength));
+                        }
+                        options.params = {
```

Comment:
I have not checked if the code works for time reasons (I have an exam tomorrow, can check afterwards), but you have probably checked this anyway, so asking is "cheaper" overall:

Does this line mean that if a user has generic parameters (any parameter other than `uptime_kuma_cachebuster`) in his*her `url`, that they are deleted or is axios smarter?

Reply:
Thankfully axios is smart here